cmd/cloud/deployment.go

```diff
@@ -359,6 +363,10 @@ func newDeploymentLogsCmd() *cobra.Command {
 	cmd.Flags().StringVarP(&logsKeyword, "keyword", "k", "", "Show logs that contain this exact keyword or phrase.")
 	cmd.Flags().IntVarP(&logCount, "log-count", "c", logCount, "Number of logs to show")
 	cmd.Flags().StringVarP(&deploymentName, "deployment-name", "n", "", "Name of the deployment to show logs of")
+	cmd.Flags().BoolVarP(&logWebserver, "webserver", "r", false, "Show logs from the webserver")
```

Comment:
We can probably get rid of those short flags since they seem to be confusing. @sunkickr Thoughts?

```suggestion
	cmd.Flags().BoolVar(&logWebserver, "webserver", false, "Show logs from the webserver")
```

Reply:
Do the same for all flags

Reply:
sounds good